proposals/0000-template.md

```diff
@@ -57,6 +57,43 @@ give a couple of examples and regard that as the specification! The
 examples should illustrate and elucidate a clearly-articulated
 specification that covers the general case.
 
+## Proposed Library Change Specification
```

Comment:
I note that the Proposed Change Specification section still has "the types and semantics of any new library interfaces" in the list of things the specification may include. Perhaps the template should make library changes a subsection (or two subsections) of the Proposed Change Specification?

```suggestion
### Library Changes
```